Pipfile

```diff
@@ -11,6 +11,7 @@ setuptools = "*"
 
 [packages]
 ifq = "*"
+pyocclient = "*"
```

Comment:
Consider set a specific package version here to have repeatable builds; you can pick the current one from the output of `pipenv graph`.

```bash
$ pipenv graph
ifq==0.2.0
  - lxml [required: >=4.5.0, installed: 4.5.0]
  - requests [required: >=2.23.0, installed: 2.23.0]
    - certifi [required: >=2017.4.17, installed: 2019.11.28]
    - chardet [required: >=3.0.2,<4, installed: 3.0.4]
    - idna [required: >=2.5,<3, installed: 2.9]
    - urllib3 [required: >=1.21.1,<1.26,!=1.25.1,!=1.25.0, installed: 1.25.8]
pyocclient==0.4
  - requests [required: >=2.0.1, installed: 2.23.0]
    - certifi [required: >=2017.4.17, installed: 2019.11.28]
    - chardet [required: >=3.0.2,<4, installed: 3.0.4]
    - idna [required: >=2.5,<3, installed: 2.9]
    - urllib3 [required: >=1.21.1,<1.26,!=1.25.1,!=1.25.0, installed: 1.25.8]
  - six [required: Any, installed: 1.14.0]
```